plugins/scaffolder-backend/api-report.md

```diff
@@ -381,6 +381,8 @@ export class DatabaseTaskStore implements TaskStore {
   // (undocumented)
   heartbeatTask(taskId: string): Promise<void>;
   // (undocumented)
+  list(options: Partial<SerializedTask>): Promise<SerializedTask[]>;
```

Comment:
is there a good reason for using SerializedTask other than not having to define a new type? `SerializedTask` include things like `id` which I guess don't make sense to list by